spark/src/main/resources/error/delta-error-classes.json

```diff
@@ -1117,7 +1117,9 @@
   },
   "DELTA_INVALID_CHARACTERS_IN_COLUMN_NAMES" : {
     "message" : [
-      "Found invalid character(s) among ' ,;{}()\\n\\t=' in the column names of your schema. Please use other characters and try again."
+      "Found invalid character(s) among ' ,;{}()\\n\\t=' in the column names of your schema.",
+      "Invalid column names: <invalidColumnNames>.",
+      "Please use other characters and try again."
```

Comment:
Should we instead suggest the user to enable column mapping on their table?

Reply:
Sounds good